src/components/RegisterForm/register-form.js

```diff
@@ -52,10 +52,6 @@ const RegisterForm = observer(
         errorText = "Passwords do not match";
         registerErrorMessage = true;
       }
-      signUpStore.email = "";
-      signUpStore.password = "";
-      signUpStore.repeatPassword = "";
-
     }
```

Comment:
This fixes the issue when the registration fails, but those fields should be cleaned every time the registration form is shown.
Maybe a method 'clear' in the signUpStore with the 'action' decorator (because it updates observable values)

Reply:
I will take a look!